packages/components/src/util/AppURLResolver.ts

```diff
@@ -60,7 +60,7 @@ export class AssayResolver implements AppRouteResolver {
                 return fetchProtocol(assayRowId)
                     .then((assay: AssayProtocolModel) => {
                         const context = {
-                            name: assay.name,
+                            name: encodeURIComponent(assay.name),
```

Comment:
I wonder if this fixes https://www.labkey.org/home/Developer/issues/issues-details.view?issueId=38907 as well.